spec/system/visit_sign_in_spec.rb

```diff
@@ -111,4 +111,25 @@
     expect(page.get_rack_session.has_key?(:challenge_parameters)).to eql false
   end
 
+    scenario 'User get propted to setup MFA on first sign in' do
```

Comment:
```suggestion
    scenario 'User get prompted to setup MFA on first sign in' do
```

and indentation below is off